streampipes-pipeline-management/src/main/java/org/apache/streampipes/manager/file/FileManager.java

```diff
@@ -104,6 +107,17 @@ public static InputStream cleanFile(InputStream fileInputStream, String filetype
     return fileInputStream;
   }
 
+  public static boolean checkFileContentChanged(String filename, String hash) throws IOException {
+    var fileHash = getFileHash(filename);
+    return !fileHash.equals(hash);
+  }
+
+  private static String getFileHash(String filename) throws IOException {
```

Comment:
This is required to be the same implementation as in `FileProtocolUtils`.
Do you see a way where we can define the method only once?
If this is not possible, I would like to some notes that make this interdependence explicit.